addons/backpacks/bussole/ui/bussole.hpp

```diff
@@ -0,0 +1,245 @@
+        h = 0.0286 * safezoneH;
+        tooltip = ECSTRING(core,rotator_volume);
+        onMouseButtonDown = "[_this select 1, true] call TFAR_fnc_setVolumeViaDialog;";
+    };
+    class stereo: HiddenButton {
+        idc = IDC_BUSSOLE_STEREO;
+        x = 0.315922 * safezoneW + safezoneX;
+        y = 0.5605 * safezoneH + safezoneY;
+        w = 0.0211406 * safezoneW;
+        h = 0.0308 * safezoneH;
+        action = "[TF_lr_dialog_radio, ((TF_lr_dialog_radio call TFAR_fnc_getCurrentLrStereo) + 1) mod TFAR_MAX_STEREO] call TFAR_fnc_setLrStereo;[TF_lr_dialog_radio] call TFAR_fnc_showRadioVolume;";
+        tooltip = ECSTRING(core,stereo_settings);
+    };
+    class additional: HiddenButton {
+        idc = IDC_BUSSOLE_ADDITIONAL;
+        x = 0.469062 * safezoneW + safezoneX;
+        y = 0.4747 * safezoneH + safezoneY;
+        w = 0.0216562 * safezoneW;
+        h = 0.0297 * safezoneH;
+        tooltip = ECSTRING(core,set_additional);
+        action = "[TF_lr_dialog_radio,TF_lr_dialog_radio call TFAR_fnc_getLrChannel] call TFAR_fnc_setAdditionalLrChannel; call TFAR_fnc_updateLRDialogToChannel; [TF_lr_dialog_radio, true] call TFAR_fnc_showRadioInfo;";
+    };
+    class speakers: HiddenButton {
+        idc = IDC_BUSSOLE_SPEAKERS;
+        x = 0.440187 * safezoneW + safezoneX;
+        y = 0.4769 * safezoneH + safezoneY;
+        w = 0.0216562 * safezoneW;
+        h = 0.0275 * safezoneH;
+        tooltip = ECSTRING(core,speakers_settings_true);
+        action = "TF_lr_dialog_radio call TFAR_fnc_setLrSpeakers;[TF_lr_dialog_radio] call TFAR_fnc_showRadioSpeakers;";
+    };
+    class channel_01: HiddenButton {
+        idc = IDC_BUSSOLE_CHANNEL_01;
+        x = 0.345828 * safezoneW + safezoneX;
+        y = 0.4747 * safezoneH + safezoneY;
+        w = 0.0216562 * safezoneW;
+        h = 0.0297 * safezoneH;
+        action = "[TF_lr_dialog_radio, 0] call TFAR_fnc_setLrChannel; [""CH%1""] call TFAR_fnc_updateLRDialogToChannel;[TF_lr_dialog_radio, true] call TFAR_fnc_showRadioInfo;";
+        tooltip = ECSTRING(core,radio_channel_1);
+    };
+    class channel_02: HiddenButton {
+        idc = IDC_BUSSOLE_CHANNEL_02;
+        x = 0.37625 * safezoneW + safezoneX;
+        y = 0.4758 * safezoneH + safezoneY;
+        w = 0.0216562 * safezoneW;
+        h = 0.0297 * safezoneH;
+        action = "[TF_lr_dialog_radio, 1] call TFAR_fnc_setLrChannel; [""CH%1""] call TFAR_fnc_updateLRDialogToChannel;[TF_lr_dialog_radio, true] call TFAR_fnc_showRadioInfo;";
+        tooltip = ECSTRING(core,radio_channel_2);
+    };
+    class channel_03: HiddenButton {
+        idc = IDC_BUSSOLE_CHANNEL_03;
+        x = 0.40925 * safezoneW + safezoneX;
+        y = 0.4758 * safezoneH + safezoneY;
+        w = 0.0216562 * safezoneW;
+        h = 0.0297 * safezoneH;
+        action = "[TF_lr_dialog_radio, 2] call TFAR_fnc_setLrChannel; [""PRE %1""] call TFAR_fnc_updateLRDialogToChannel;[TF_lr_dialog_radio, true] call TFAR_fnc_showRadioInfo;";
```

Comment:
Other channels use `[""CH%1""] call TFAR_fnc_updateLRDialogToChannel` Why does this and the next once suddenly use ``[""PRE %1""] call TFAR_fnc_updateLRDialogToChannel

Reply:
Actually.. https://github.com/michail-nikolaev/task-force-arma-3-radio/blob/1.0/addons/core/functions/events/ui/fnc_updateLRDialogToChannel.sqf#L24
Don'T need that param at all.